ex02/ex22.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+input="$@"
+
+if [[ -n "$input" ]]; then
+  finded=$(find -name $input)
+  output=$(ls -Rd $finded)
+  echo $input
+  echo "$output"
+fi  
```

Comment:
@Esknor you can simplify this. Don't use redundant local variables

```bash
#!/bin/bash
if [[ -n "$@" ]]; then
  echo $@
  finded=$(find -name $@)
  output=$(ls -Rd $finded)
  echo "$output"
fi
```

Reply:
ok